ProcessLib/LiquidFlow/LiquidFlowMaterialProperties.h

```diff
@@ -79,10 +102,12 @@ class LiquidFlowMaterialProperties
 
     double getViscosity(const double p, const double T) const;
```

Comment:
I thought that this class only collects porous media material properties and `LiquidFlowMaterialProperties` is responsible for viscosity and density. So probably my understanding is not correct. So my maby stupid question: Why is the viscosity in this class?